server.go

```diff
@@ -47,6 +52,25 @@ func NewServer(ctx context.Context) *Server {
 	}
 	s.DB = pool
 
+	// Set up the in-memory cache
+	memcached, err := memory.NewAdapter(
+		memory.AdapterWithAlgorithm(memory.LRU),
+		memory.AdapterWithCapacity(int(math.Pow(1000, 3))), // 1000^3 = gigabyte
+	)
+	if err != nil {
+		log.Fatal("error setting up memory cache:", err)
+	}
+
+	cacheClient, err := cache.NewClient(
```

Comment:
@qtrinh2 What do you think of these settings in particular? It caches for 1 hour and uses 1 GB of memory for the cache.